### DISCUSSION TITLE
如何修改公告的显示超时时间

### DISCUSSION THREAD

**Discussion by auqhjjqdo0** [AlistGo/alist#2245] [General]
现在公告的默认显示时间只有3秒，对于稍长一点的公告3秒可能看不完，有没有什么方法可以修改公告的显示时间

**Comment by 12189108** [comment]
临时解决方案:换成md文件显示？

**Reply by auqhjjqdo0** [reply]
> 临时解决方案:换成md文件显示？

那作者就没有必要出个公告的功能了
弹窗公告的目的就是为了强调某些内容，什么都写进readme很多人并不想看
